src/webhook/mutation/pod_mutator.go

```diff
@@ -296,13 +296,13 @@ func addToInitContainers(pod *corev1.Pod, installContainer corev1.Container) {
 	pod.Spec.InitContainers = append(pod.Spec.InitContainers, installContainer)
 }
 
-func updateContainers(pod *corev1.Pod, injectionInfo *InjectionInfo, ic *corev1.Container, dk dynatracev1beta1.DynaKube, deploymentMetadata *deploymentmetadata.DeploymentMetadata) {
+func (m *podMutator) updateContainers(pod *corev1.Pod, injectionInfo *InjectionInfo, ic *corev1.Container, dk dynatracev1beta1.DynaKube, deploymentMetadata *deploymentmetadata.DeploymentMetadata, nsName string) {
```

Comment:
Since another parameter has been added, and the signature was too long before already, please refactor your code such that parameter list does not have more than two parameters.

Reply:
@meik99 I've done a slight refactor which sort of proliferated throughout the pod_mutator code. I've added two new fields to the `podMutator` struct for the `currentPodNS` and also for  the current/active `dynaKube` instance related to the incoming pod request. Please review.